features/people.feature

```diff
@@ -65,4 +65,9 @@ Scenario: Images are virus-checked before publication
   When I add a new person called "Dave Cameroon"
   Then the image will be quarantined for virus checking
   When the image has been virus-checked
-  Then the virus checked image will be available for viewing
\ No newline at end of file
+  Then the virus checked image will be available for viewing
+
+Scenario: Viewing a person that previously had a role
+  Given "Dale Cooper" is a minister with a history
+  When I visit the person page for "Dale Cooper"
+  Then I should see past information about the person "Dale Cooper"
```

Comment:
Does it matter that this step is actually testing more than just the truncated biography?